src/price/ProgressivePriceThresholdsDto.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace hiqdev\php\billing\price;
+
+use Money\Currency;
+
+class ProgressivePriceThresholdsDto
```

Comment:
DTO is used to pass a big set of options required to build an object in Factory... when you don't want to carry data in arrays.

For ProgressivePriceThreshold it makes more sense to add a named constructor that will create some missing objects, than having DTO + Factory.